tests/testthat/test-estimate_truncation.R

```diff
@@ -6,39 +6,8 @@ futile.logger::flog.threshold("FATAL")
 old_opts <- options()
 options(mc.cores = ifelse(interactive(), 4, 1))
 
-# get example case counts
-reported_cases <- example_confirmed[1:60]
-
-# define example truncation distribution (note not integer adjusted)
-trunc_dist <- dist_spec(
-  mean = convert_to_logmean(3, 2),
-  mean_sd = 0.1,
-  sd = convert_to_logsd(3, 2),
-  sd_sd = 0.1,
-  max = 10
-)
-
-# apply truncation to example data
-construct_truncation <- function(index, cases, dist) {
-  set.seed(index)
-  cmf <- cumsum(
-    dlnorm(
-      1:(dist$max + 1),
-      rnorm(1, dist$mean_mean, dist$mean_sd),
-      rnorm(1, dist$sd_mean, dist$sd_sd)
-    )
-  )
-  cmf <- cmf / cmf[dist$max + 1]
-  cmf <- rev(cmf)[-1]
-  trunc_cases <- data.table::copy(cases)[1:(.N - index)]
-  trunc_cases[(.N - length(cmf) + 1):.N, confirm := as.integer(confirm * cmf)]
-  return(trunc_cases)
-}
-example_data <- purrr::map(c(20, 15, 10, 0),
-  construct_truncation,
-  cases = reported_cases,
-  dist = trunc_dist
-)
+# get example truncated case counts
+example_data <- example_truncated
```

Comment:
This is not needed. Just apply the tests directly to `example_truncated` below.

Reply:
Isn't this one still open?

Reply:
Strange! I fixed it here 793ae679ef61576de9f28410d0535cf432c25115.

Reply:
Sorry, I was looking at a different file.